.github/workflows/pypi_publish.yml

```diff
@@ -0,0 +1,64 @@
+name: Build Wheels and upload to PyPI
+
+on:
+  pull_request:
+    branches:
+      - "releases/**"
+    types: [labeled, opened, synchronize, reopened]
+  release:
+    types: [published]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  build_wheels_sdist:
+    name: Build
+    uses: ./.github/workflows/build.yml
+    secrets: inherit
+
+  test_pypi_publish:
```

Comment:
Is this the version that was previously in alpineer that will work, but won't permit multiple test_pypi publishes? since we're not using poetry here?

Reply:
It should work as well here. I'm using `setuptools_scm` which has the functionality to add `.dev+{number of commits}` appended to the next version. It can also be ran locally with `python -m build` to simulate what `cibuildwheel` does.

Reply:
Just ran `python -m build`, and I get the following output:
```sh
Successfully built ark-analysis-0.5.0.dev122.tar.gz and ark_analysis-0.5.0.dev122-cp38-cp38-macosx_13_0_arm64.whl
```

This format is uploadable to Test PyPI. The reason the version scheme is 0.5.0 is cause that's the most recent version on [PyPI](https://pypi.org/project/ark-analysis/). Therefore, this current branch is 122 commits ahead of version 0.5.0.

Reply:
Sounds good. Let's keep all of the releases on 0.5.x, I'm guessing we may need a couple more to sort everything out. Then when we're ready for Candace's paper, can do a final 0.6.0